modules/tinymce/src/plugins/table/test/ts/browser/command/TableSizingModeCommandUtil.ts

```diff
@@ -0,0 +1,70 @@
+import { Log } from '@ephox/agar';
```

Comment:
This test file is in the wrong place, it's not a test and instead is a helper module. As we mentioned during the call, it should be in `src/test/ts/module/`

Reply:
This is still slightly in the wrong location sorry. It should be beside the `TableTestUtils` file.